spec/factories/in_person_enrollments.rb

```diff
@@ -42,8 +43,8 @@
       association :notification_phone_configuration
     end
 
-    trait :with_sponsor_id do
```

Comment:
I deleted this trait now that it comes on :in_person_enrollment - see line 8 above

Reply:
Makes sense to me. 👍🏻